update-lit.sh

```diff
@@ -0,0 +1,155 @@
+#!/bin/bash
```

Comment:
Please make this use `/bin/sh`, if it needs bash features then find an alternative to those features.
Also: use `set -eu` to prevent errors from propagating